com.unity.netcode.gameobjects/Runtime/Simulator/NetworkSimulator.cs

```diff
@@ -1,13 +1,13 @@
 ﻿using Unity.Netcode.Transports.UTP;
 using UnityEngine;
-using UnityEngine.Serialization;
 
 namespace Unity.Netcode
 {
     [RequireComponent(typeof(UnityTransport))]
     public class NetworkSimulator : MonoBehaviour
     {
-        private NetworkSimulatorConfiguration m_SimulatorConfiguration;
+        [SerializeField]
+        NetworkSimulatorConfiguration m_SimulatorConfiguration;
```

Comment:
Missing the private keyword.

Reply:
I still didn't figure out what's the correct style guide to follow, according to [this doc](https://confluence.unity3d.com/pages/viewpage.action?spaceKey=DEV&title=Coding+Style+Guidelines) with [this reference](https://ono.unity3d.com/unity-extra/unity-meta/raw/@/ReferenceSource/CSharp/Assets/CSharpReference.cs) we should _"Drop redundant access specifiers"_ .

Reply:
Yeah we should. NGO however is not exactly following. It's confusing, I know.